OPCB/initServer.sqf

```diff
@@ -1,76 +1,97 @@
-call compile preprocessFileLineNumbers "unitTypes.sqf";
+enableSaving [false, false];
+enableSentences true;
+enableTeamswitch false;
 
-["Initialize"] call BIS_fnc_dynamicGroups;
-call compileFinal preprocessfilelinenumbers "functions\BADCO_skin_applier.sqf";
-call compileFinal preprocessfilelinenumbers "Scripts\BADCO_Arsenal.sqf";
+#include "unitTypes.sqf";
+
+call compileFinal preprocessFileLineNumbers "economy\init.sqf";
 
-missionNamespace setVariable ["running_task",0];
-missionNamespace setVariable ["task_spot",[5840,5700,0]];
+[] execVM "Scripts\ied.sqf";
+[] execVM "insurgency\init.sqf";
+[] execVM "Vcom\VcomInit.sqf";
 
-missionNamespace setVariable ["enemy_groups",[]];
+Resistance setFriend [EAST, 1]; Resistance setFriend [WEST, 0]; Resistance setFriend [Civilian, 1];
 
-missionNamespace setVariable ["Chapo_trigger",false];
+EAST setFriend [Resistance, 1]; EAST setFriend [WEST, 0]; EAST setFriend [Civilian, 1];	
 
+WEST setFriend [EAST, 0]; WEST setFriend [Resistance, 0]; WEST setFriend [Civilian, 1]; 	
 
-missionNamespace setVariable ["MaxTanks",0,true];
-missionNamespace setVariable ["MaxAttackHelis",0,true];
-missionNamespace setVariable ["MaxTransHelis",0,true];
-missionNamespace setVariable ["MaxAPC",0,true];
-missionNamespace setVariable ["MaxStatic",0,true];
+Civilian setFriend [EAST, 1]; Civilian setFriend [WEST, 1]; Civilian setFriend [Resistance, 1];
 
-_zeus_group = createGroup sideLogic;
+Hz_pers_var_insurgencyClearedMarkers = [];
+Hz_pers_customLoadFunction = compileFinal preprocessFileLineNumbers "Hz_pers_customLoadFunction.sqf";
+Hz_pers_firstTimeLaunchFunction = compileFinal preprocessFileLineNumbers "Hz_pers_firstTimeLaunchFunction.sqf";	
 
-missionNamespace setVariable ["Zeus_group",_zeus_group];
+["Initialize"] call BIS_fnc_dynamicGroups;
+#include "functions\BADCO_Arsenal.sqf"
+
+IsATaskRunning = false;
+TaskSpot = [5840,5700,0];
+TaskNumber = 0;
+EnemyGroups = [];
+ChapoTrigger = false;
+
+CrateCount = 0;
+MaxTanks = 0;
+MaxAttackHelis = 0;
+MaxTransHelis = 0;
+MaxAPC = 0;
+MaxStatic = 0;
+
+addMissionEventHandler ["OnUserSelectedPlayer", {
```

Comment:
What is the purpose of this? Are those vars already not `publicvariable`'d every time they're changed?